pytest_odoo.py

```diff
@@ -45,7 +48,7 @@ def pytest_cmdline_main(config):
         options = []
         # Replace --odoo-<something> by --<something> and prepare the argument
         # to propagate to odoo.
-        for option in ['--odoo-database', '--odoo-log-level']:
+        for option in ['--odoo-database', '--odoo-log-level', '--odoo-config']:
```

Comment:
I think you have to include this in the main IF above: if you pass a config file unlikely you are goinf to pass the rest of the vars because you'll have all in one. Am I wrong?